pkgs/tools/misc/uhk-agent/default.nix

```
@@ -0,0 +1,44 @@
+{ lib, stdenv, bash, appimage-run }:
+let
+  uhk-agent-bin = stdenv.mkDerivation rec {
+    pname = "uhk-agent-bin";
+    version = "1.5.16";
+    src = builtins.fetchurl {
```

```suggestion
{ lib, stdenv, fetchurl, bash, appimage-run }:
let
  uhk-agent-bin = stdenv.mkDerivation rec {
    pname = "uhk-agent-bin";
    version = "1.5.16";
    src = fetchurl {
```
The fixed-output derivation fetcher should not be preferred over the builtin. Note that this is why the CI is complaining